blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json

```diff
@@ -375,31 +411,34 @@
           }
         },
         "width": 6,
-        "yPos": 20
+        "xPos": 6,
+        "yPos": 16
       },
       {
         "height": 4,
         "widget": {
-          "title": "ip_addresses_per_subnet_utilization",
+          "title": "firewalls_per_project_usage",
```

Comment:
This should be firewalls_per_project_vpc_usage

Reply:
y